frame/support/src/lib.rs

```diff
@@ -83,9 +90,21 @@ pub mod s2s {
 		result.copy_from_slice(&encoded);
 		result
 	}
+
+	// 128 bit or 16 bytes
+	// [0..4]  bytes ---- reserved
+	// [4..8]  bytes ---- laneID
+	// [8..16] bytes ---- message nonce
+	pub fn nonce_to_message_id(lane_id: &[u8], nonce: u64) -> BridgeMessageId {
+		let mut message_id: BridgeMessageId = Default::default();
+		message_id[4..8].copy_from_slice(&lane_id[..4]);
+		message_id[8..].copy_from_slice(&nonce.to_be_bytes());
+		message_id
+	}
 }
 
 pub type PalletDigest = [u8; 4];
+pub type BridgeMessageId = [u8; 16];
```

Comment:
Docs require.

```
/// [0..4]  bytes ---- reserved
/// [4..8]  bytes ---- laneID
/// [8..16] bytes ---- message nonce
```

Reply:
Why not keeping the message-id format consistent with parity-bridges-common, `(lane_id, message nonce)`. What is the purpose of the reserved bytes?

Reply:
Maybe this definition is not so good. But we should not use `(lane_id, message nonce)` directly as in parity-bridges-common. We need a common definition of `message id`. And this `message id` should be generic for sub<>sub， ethereum<>darwina， bsc<>darwinia, etc.
The reserved is not used now. Maybe it will be filled by chainID or some other identities.